pkgs/by-name/ra/railsy/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "railsy";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "mmkaram";
+    repo = "railsy";
+    rev = "ref/tags/v${version}";
```

Comment:
When I tried building this, there is no such tag in the mmkaram/railsy repo, you'd need to add it, or use a commit as the `rev`